src/libs/CollectionUtils.ts

```diff
@@ -0,0 +1,10 @@
+function lastItem<T>(object: Record<string, T> = {}): T {
+    const lastKey = Object.keys(object).pop() ?? 0;
```

Comment:
I think the `?? 0` can be removed 🤔 It does not safeguard against anything, since the only case it might be useful would be if the object was empty, but if the object was empty, the return value of the function will be undefined, no matter if we set 0 or not.